### ISSUE TITLE
Spacy en_core_web_md error

### ISSUE THREAD

**Issue by dhwani2410** [explosion/spaCy#4566]
## Command
```
 nlp =  spacy.load("en_core_web_md")

abc= "Trypsin-sensitive photosynthetic activities in chloroplast membranes from Chlamydomonas reinhardi, y-1. Location of electron transport chain components in chloroplast membranes of chlamydomonas reinhardi, y-1 was investigated by use of proteolytic digestion with soluble or insolubilized trypsin. Digestion of intact membrane vesicles with soluble trypsin inactivates the water-splitting system, the 3-(3,4-dichlorophenyl)-1,1-dimethylurea inhibition site of Photosystem II, the electron transport between the two photosystems as well as the ferredoxin NADP reductase. Reduction of NADP with artificial electron donors for Photosystem I could be restored, however, by addition of purified reductase to trypsin-digested membranes. Electron transfer activities of Photosystems I and II reaction centers were resistant to trypsin digestion either from outside or from within the thylakoids when active trypsin was trapped inside the membrane vesicles by sonication and digestion carried out in the presence of trypsin inhibitor added from outside. In the latter case, the water-splitting system was also found to be resistant to digestion. Polyacrylamide-bound insolubilized trypsin inactivated only the ferredoxin NADP reductase. Photosynthetically active membranes obtained at different stages of development showed a basically similar behavior toward trypsin"

nlp(abc)

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dhwani.dholakia/anaconda3/lib/python3.7/site-packages/spacy/language.py", line 438, in __call__
    doc = proc(doc, **component_cfg.get(name, {}))
  File "pipes.pyx", line 393, in spacy.pipeline.pipes.Tagger.__call__
  File "pipes.pyx", line 438, in spacy.pipeline.pipes.Tagger.set_annotations
  File "morphology.pyx", line 312, in spacy.morphology.Morphology.assign_tag_id
  File "morphology.pyx", line 200, in spacy.morphology.Morphology.add
ValueError: [E167] Unknown morphological feature: 'ConjType' (9141427322507498425). This can happen if the tagger was trained with a different set of morphological features. If you're using a pretrained model, make sure that your models are up to date:
python -m spacy validate
```

## Info about spaCy

* **spaCy version:** 2.2.1
* **Platform:** Linux-2.6.32-279.el6.x86_64-x86_64-with-redhat-6.7-Santiago
* **Python version:** 3.7.4

**Comment by lneeraj97** [comment]
Found this https://github.com/allenai/scispacy/issues/167

Apparently, downgrading your spacy version fixes it.

**Comment by adrianeboyd** [comment]
You can check your installed models with `spacy validate` and it will let you know if the models aren't compatible with the version of spacy you are currently using. (You are probably trying to load a v2.2 model with spacy v2.1.)

It looks like scispacy just released a new version with models for v2.2, but it might not be stable yet. If you are using scispacy, you could try downgrading scispacy to the previous version with spacy v2.1.9.

**Comment by ines** [comment]
If the models are not up to date: upgrade them. If `spacy validate` shows that you're using the latest version, try redownloading them (see #4568).
